src/app/view/home/home.component.ts

```diff
@@ -6,18 +6,20 @@ import { Component, OnInit } from '@angular/core';
     styleUrls: ['./home.component.scss']
 })
 export class HomeComponent implements OnInit {
+    public honkAudio1 = new Audio('../../../assets/Honk1.wav');
```

Comment:
Reason why I made it `1` is because I was thinking of loading more than one sound, and then randomizing it later. so you get a different Hjönk every time.

Reply:
I have more HJÖNKS. What format is most optimal for web scale HJÖNKING?

Reply:
wave files are usually bigger than mp3. let's see if my change fix the mobile thing first.